pkg/cloudprovider/providers/azure/azure_standard.go

```diff
@@ -415,6 +415,21 @@ func (as *availabilitySet) GetZoneByNodeName(name string) (cloudprovider.Zone, e
 		return cloudprovider.Zone{}, err
 	}
 
+	// Get availability zone for the node.
```

Comment:
would you also update the `GetZoneByNodeName` func comments?

Reply:
ack, yep